config/config/test/fixtures/settings-in-workspace-yaml/pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages: []
+onlyBuiltDependencies:
+  - foo
```

Comment:
How to express things like [overrides](https://pnpm.io/package_json#pnpmoverrides) in yaml? Perhaps more examples from that doc should be written as test.

```
{
  "pnpm": {
    "overrides": {
      "foo": "^1.0.0",
      "quux": "npm:@myorg/quux@^1.0.0",
      "bar@^2.1.0": "3.0.0",
      "qar@1>zoo": "2"
    }
  }
}
```

Reply:
To be safe, you can keep double quotes:

```yaml
    "overrides":
      "foo": "^1.0.0"
      "quux": "npm:@myorg/quux@^1.0.0"
      "bar@^2.1.0": "3.0.0"
      "qar@1>zoo": "2"
```